src/test/java/org/folio/rest/impl/MockServer.java

```diff
@@ -953,6 +956,42 @@ private void handleGetGenericSubObj(RoutingContext ctx, String subObj) {
     }
   }
 
+  private void handleGetPieceById(RoutingContext ctx) {
+
+    logger.info("handleGetPiecesById got: " + ctx.request()
+      .path());
+    String pieceId = ctx.request().getParam(ID);
+    JsonObject body;
+    try {
+      if (NON_EXIST_VENDOR_ID.equals(pieceId)) {
+        serverResponse(ctx, HttpStatus.HTTP_NOT_FOUND.toInt(), APPLICATION_JSON, "vendor not found");
+      } else if (MOD_VENDOR_INTERNAL_ERROR_ID.equals(pieceId)) {
+        serverResponse(ctx, HttpStatus.HTTP_INTERNAL_SERVER_ERROR.toInt(), APPLICATION_JSON,
+            "internal server error, contact administrator");
+      } else if (PIECE_POLINE_CONSISTENCY_404_POLINE_NOT_FOUND_ID.equals(pieceId)) {
+        body = new JsonObject(ApiTestBase
+          .getMockData(PIECE_RECORDS_MOCK_DATA_PATH + "pieceRecord-poline-not-exists-5b454292-6aaa-474f-9510-b59a564e0c8d.json"));
+        if (body != null) {
+          serverResponse(ctx, HttpStatus.HTTP_OK.toInt(), APPLICATION_JSON, body.encodePrettily());
+        }
+      } else if (PIECE_POLINE_CONSISTENT_RECEIPT_STATUS_ID.equals(pieceId)) {
+        body = new JsonObject(ApiTestBase.getMockData(
+            PIECE_RECORDS_MOCK_DATA_PATH + "pieceRecord-received-consistent-receipt-status-5b454292-6aaa-474f-9510-b59a564e0c8d2.json"));
+        if (body != null) {
```

Comment:
The `body` can never be `null` because of `new JsonObject(<file_content>)`

Reply:
Removed null check